### PR TITLE
[azure] auto case OCP-24122

### PR DESCRIPTION
**Pull request by jinyunma**
auto case OCP-24122: [ipi-on-azure] Try to create multiple clusters with the same Cluster Name + the same base domain

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.18-azure-ipi-valid-duplicate-dns-f14

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.18-azure-ipi-valid-duplicate-dns-f14

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.18-azure-ipi-valid-duplicate-dns-f14

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.18-azure-ipi-valid-duplicate-dns-f14

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml [hunk 1]
```diff
@@ -299,6 +299,13 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-private-sshkey
+- as: azure-ipi-valid-duplicate-dns-f14
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml:302]: Can we use arm arch to run the testing ?

Comment (jinyunma) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml]: sure, updated.

**Comment by jinyunma** [comment]
The step `cucushift-installer-rehearse-azure-cases-valid-duplicate-dns-provision` is passed in [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58224/rehearse-58224-periodic-ci-openshift-verification-tests-master-installation-nightly-4.18-azure-ipi-valid-duplicate-dns-f14/1851092932996632576).

/pj-rehearse ack

@jianlinliu PTAL again, thanks.

**Pull request merged**